src/olympia/scanners/tasks.py

```diff
@@ -238,8 +237,10 @@ def mark_yara_query_rule_as_completed_or_aborted(query_rule_pk):
     rule = ScannerQueryRule.objects.get(pk=query_rule_pk)
     try:
         if rule.state == RUNNING:
+            log.info('Marking Yara Query Rule %s as completed', rule.pk)
```

Comment:
these two `log.info` statements seem unrelated?

Reply:
Yeah, it's just additional logging that I slipped in because you can never have enough logging :)